docs/v2/payroll_au/index.html

```diff
@@ -3411,7 +3411,7 @@
           <nav id="scrollingNav">
             <ul class="sidenav nav nav-list">
               <li class="nav-header" data-group="PayrollAu"><strong>SDK: </strong><span id='sdk-name'></span></li>
-              <li class="nav-header" data-group="PayrollAu"><strong>VSN: </strong>2.23.2</li>
+              <li class="nav-header" data-group="PayrollAu"><strong>VSN: </strong>2.25.0</li>
```

Comment:
version is wrong here

Reply:
Thanks for pointing. Updated the version.